articles/hosted-pages/password-reset.md

```diff
@@ -17,6 +17,23 @@ Using the [Auth0 Dashboard](${manage_url}/#/password_reset), you can enable your
 
 Once you've enabled the Password Reset Page for your account, you'll be able to use the text editor built into the Auth0 Dashboard to change your HTML, style your page using CSS, and alter the JavaScript used to retrieve custom variables. After you've made your changes, and make sure to click _Save_.
 
+### Custom Variables
```

Comment:
That looks good Katie. I'm wondering though, wouldn't it be better if we added one example? The script for a passwd reset page retrieving and displaying a couple of these variables?